apollo-client/src/main/java/com/ctrip/framework/apollo/spring/config/PropertySourcesProcessor.java

```diff
@@ -82,17 +82,20 @@ private void initializePropertySources() {
     // clean up
     NAMESPACE_NAMES.clear();
 
-    // add after the bootstrap property source or to the first
-    if (environment.getPropertySources()
-        .contains(PropertySourcesConstants.APOLLO_BOOTSTRAP_PROPERTY_SOURCE_NAME)) {
+    // ensure ApolloBootstrapPropertySources is still the first
+    ensureBootstrapPropertyPrecedence(environment);
 
-      // ensure ApolloBootstrapPropertySources is still the first
-      ensureBootstrapPropertyPrecedence(environment);
+    if (composite.getPropertySources() != null && composite.getPropertySources().size() > 0) {
```

Comment:
how about using `return early pattern` which keeps the code a little bit clean?

```java
  if (CollectionUtils.isEmpty(composite.getPropertySources())) {
      return;
  }
```

Reply:
it seems a good advice.

Reply:
It looks good to me. However, the code style is not the same, e.g. Apollo's code style uses 2 spaces as indent, would you please use [apollo-code-style](https://github.com/ctripcorp/apollo/tree/master/apollo-buildtools/style) and format the changes?

Reply:
i have format the code using the [apollo-code-style](https://github.com/ctripcorp/apollo/tree/master/apollo-buildtools/style) you provide